pytest_func_cov/plugin.py

```diff
@@ -152,6 +155,22 @@ def pytest_terminal_summary(self, terminalreporter):
 
         args = ("TOTAL", total_funcs, total_miss, total_cover)
 
+        if include_json:
+            report_data = {
+                "total_funcs": total_funcs,
+                "total_miss": total_miss,
+                "total_cover": total_cover,
+            }
+
+            try:
+                file_path = os.path.join(self.json_path[0], "func_report.json")
+                os.makedirs(self.json_path[0], exist_ok=True)
```

Comment:
It would be better if you made the config ini include the full path to the file to output instead of just the directory.